src/_premake_init.lua

```diff
@@ -1655,6 +1655,9 @@
 
 	-- Add variations for other Posix-like systems.
 
+	filter { "system:MacOSX", "kind:WindowedApp" }
+		targetextension ".app"
```

Comment:
Could this also apply to `system:ios`?

Reply:
I did not specifically think about `system: ios`.
Probably it is better to apply.

Reply:
Even if `_TARGET_OS` is set to `ios`, the generated project seems to be correct, but should I modify something by myself?
(e.g. `fliter { "system:macosx or ios", "kind:WindowedApp" }`)